tools/packaging/static-build/qemu/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN apt-get update && apt-get upgrade -y && \
 	    libpixman-1-dev${DPKG_ARCH} \
 	    libselinux1-dev${DPKG_ARCH} \
 	    libtool${DPKG_ARCH} \
+	    liburing-dev${DPKG_ARCH} \
```

Comment:
@zvonkok - it looks this should go along with the ubuntu bump.

Reply:
You mean in another commit?

Reply:
Yes, this should go along with the removal of the manual build of liburing, either in the commit that bumps ubuntu or a subsequent commit.